OvmfPkg/Library/FdtNorFlashQemuLib/FdtNorFlashQemuLib.c

```diff
@@ -0,0 +1,165 @@
+/** @file
+
+  Copyright (c) 2014-2018, Linaro Ltd. All rights reserved.<BR>
+  Copyright (c) 2024 Loongson Technology Corporation Limited. All rights reserved.<BR>

```

Comment:
Which part of this file is original code? If none, drop this copyright claim.

Reply:
In fact, most of the code about frame is original, only new logic is added between lines 111 to 136, and the logic of lines 100 and 101 are adjusted.